File: python/amazon.py
import requests

task_params = {
    'target': 'amazon',
    'url': 'https://www.amazon.com/dp/B09H74FXNW',
    'parse': True,
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scraper-api.smartproxy.com/v2/scrape',
    json = task_params,
    auth = (username, password)
)

print(response.text)

File: nodejs/amazon.js
const response = await fetch(
    'https://scraper-api.smartproxy.com/v2/scrape', {
        method: 'POST',
        body: {
            target: 'amazon',
            parse: true,
            url: 'https://www.amazon.com/dp/B09H74FXNW'
        },
        headers: {
            'Content-Type': 'application/json',
            'Authorization': 'Basic AUTH'
        },
    }
).catch(error => console.log(error));

console.log(response)

